x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/helper.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get, pick, isEmpty } from 'lodash';
+import { Section as SectionType } from './sections';
```

Comment:
Why is this renamed to `SectionType`. If there are naming conflicts we should consider whether the names are apt in the first place